FlowCrypt/Core/Models/DecryptedPrivateKey.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DecryptedPrivateKey.swift
+//  FlowCrypt
+//
+//  Created by Yevhen Kyivskyi on 16.07.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+
+struct DecryptedPrivateKeysContainer: Decodable {
+
+    let privateKeys: [DecryptedPrivateKey]
+}
+
+struct DecryptedPrivateKey: Decodable {
+
+    let decryptedPrivateKey: String
+}
```

Comment:
We should categorize this somehow for clarity, for example place these in ` FlowCrypt/Core/Models/EKM/EkmPrivateKeysGetResponse.swift` and rename `DecryptedPrivateKeysContainer` to `EkmPrivateKeysGetResponse`

Similarly later, we should put FES responses into their own folder and name them with similar structures. (for FES, I'll make another issue for later)